CHANGELOG.md

```diff
@@ -1,3 +1,126 @@
+### citus v11.1.0 (September 15, 2022) ###
+
+* Adds support for PostgreSQL 15beta4
+
+* Adds a rebalancer that uses background tasks for its execution
+
+* Adds `create_distributed_table_concurrently()` UDF to distribute tables
+  without blocking writes
+
+* Adds `citus_split_shard_by_split_points()` UDF that allows
+  splitting a shard to specified set of nodes without blocking writes
+  and based on given split points
+
+* Adds support for non-blocking tenant isolation
+
+* Adds support for isolation tenants that use partitioned tables
+  or columnar tables
+
+* Drops support for isolation tenants that use replicated tables
+
+* Separates columnar table access method into a separate logical extension
+
+* Adds support for logical replication in `replicate_reference_tables()`
+
+* Improves performance of blocking shard moves
+
+* Improves non-blocking shard moves with a faster custom copy logic
+
+* Adds the GUC `allow_unsafe_constraints` to allow constraints without
```

Comment:
Hmm, what does `constraints without distribution column` mean? Can you share the link to the PR that adds this?

Reply:
also, maybe `citus.allow_unsafe_constraints`

Reply:
https://github.com/citusdata/citus/pull/6237

Reply:
oh, I haven't seen this. Maybe we call it as `unique constraint`? Because, for example, we don't seem to relax foreign keys etc